src/frontend/packages/kubernetes/src/helm/list-types/monocular-chart-card/monocular-chart-card.component.ts

```diff
@@ -12,7 +16,16 @@ export class MonocularChartCardComponent extends CardCell<MonocularChart> {
 
   @Input() row: MonocularChart;
 
+  public artifactHubAndOthers$: Observable<boolean>;
+
   constructor() {
     super();
+    this.artifactHubAndOthers$ = stratosEntityCatalog.endpoint.store.getAll.getPaginationService().entities$.pipe(
```

Comment:
Can we update the variable name - this is detecting artifactHub - we always show the icon if Artifact HUb is registered - it doesn't depend on others.

Reply:
In fact, do we even need this?

When it is used, we are just checking of the endpoint it the artifact hub endpoint and using that to show the icon, so I don't think this observable is necessary.

Reply:
@nwmac I think my original intent was that we only needed to differentiate between the two types (artifact hub and normal helm repo) if there would be confusion between charts... so if there's an artifact hub endpoint and at least one normal helm repo endpoint then show something. Otherwise if it's just artifact hub or only normal helm repo's don't show anything. Minus the bugs that I put in, is that still sound? Otherwise we could just always show the aritfact hub icon, if the chart is from there, regardless of other endpoints?